protobuf/flagd/evaluation/v1/schema.proto

```diff
@@ -0,0 +1,193 @@
+/**
+ * Flag evaluation API
+ *
+ * This proto forms the basis of a flag-evaluation API.
+ * It supports single and bulk evaluation RPCs, and flags of various types, as well as establishing a stream for getting notifications about changes in a flag definition.
+ * It supports the inclusion of a "context" with each evaluation, which may contain arbitrary attributes relevant to flag evaluation.
+ */
+syntax = "proto3";
+
+package flagd.evaluation.v1;
+
+import "google/protobuf/struct.proto";
+
+option csharp_namespace = "OpenFeature.Flagd.Grpc.Evaluation";
+option go_package = "flagd/evaluation/v1";
+option java_package = "dev.openfeature.flagd.grpc.evaluation";
+option php_namespace = "OpenFeature\\Providers\\Flagd\\Schema\\Grpc\\Evaluation";
+option ruby_package = "OpenFeature::FlagD::Provider::Grpc::Evaluation";
```

Comment:
@toddbaert should we use this as an opportunity to lowercase the `D` based on the updated naming guidelines?

Reply:
yes!